lib/vyasa_web/router.ex

```diff
@@ -26,14 +26,12 @@ defmodule VyasaWeb.Router do
       on_mount: [{VyasaWeb.Session, :sangh}, {VyasaWeb.Hook.UserAgent, :default}] do
       # NOTE: SLUG PATTERN:
       # /<mode>/<binding-info-slug>/<final_identifying_slug>?=<url-encoded-params>
-      # TODO: to standardise, should rename "explore" -> "read"
-      # TODO: @ks0m1c @rtshkmr the actions need to be udpated so that mediator can pipe things based on mode
-      live "/explore/", ModeLive.Mediator, :show_sources
-      live "/explore/:source_title/", ModeLive.Mediator, :show_chapters
-      live "/explore/:source_title/:chap_no", ModeLive.Mediator, :show_verses
-      # live "/discuss/", ModeLive.Mediator, :show_sources
-      # live "/discuss/:source_title/", ModeLive.Mediator, :show_chapters
-      # live "/discuss/:source_title/:chap_no", ModeLive.Mediator, :show_verses
+      live "/read/", ModeLive.Mediator, :show_sources
+      live "/read/:source_title/", ModeLive.Mediator, :show_chapters
+      live "/read/:source_title/:chap_no", ModeLive.Mediator, :show_verses
+      live "/discuss/", ModeLive.Mediator, :index
+      live "/discuss/:source_title/", ModeLive.Mediator, :discuss_source
+      live "/discuss/:source_title/:chap_no", ModeLive.Mediator, :discuss_chapter
```

Comment:
i think we can abstract out discuss and read to :mode and mediator can default its behaviours through inject mode

Reply:
the reason i didn't do this in the first place was that having the first section dynamic will mean that the regex match will happen for ALL paths. 

I tried to follow this advice, but very soon got an example of why this shouldn't be the case:

```elixir
CHECK slot_selector_key: :mode_context_component_selector
CHECK slot_selector_key: "selector: read, slot selector_key: mode_context_component_selector"
URL PARMS:: %{
  :path => "/live/longpoll",
  "_csrf_token" => "BhE6ERwjPhB_PVoBM3BBYycxOV0_ORwCVIoc_yfHKKmoK7rQBGl2Uh17",
  "_live_referer" => "undefined",
  "_mounts" => "0",
  "_track_static" => %{
    "0" => "http://localhost:4000/assets/app.css",
    "1" => "http://localhost:4000/assets/app.js"
  },
  "locale" => "en-GB",
  "mode" => "live",
  "session" => %{
    "active" => "true",
    "id" => "IhrQV0ypqg8bHakbvb2bymNF",
    "last_opened" => "2024-09-23T11:03:32.022987Z",
    "name" => "ritesh kumar",
    "sangh" => %{"id" => "164eb05d-221a-4939-b180-8394e1a5515f"}
  },
  "source_title" => "longpoll",
  "timezone" => "Asia/Singapore",
  "timezone_offset" => "480",
  "vsn" => "2.0.0"
}
```

### Middle ground

hence, I'll do a middle ground: 

1. I'll put a static prefix in the slug so the pattern for urls that the mediator manages is `/m/<mode>/<source_title>/<chap_no>`